android/app/src/main/res/layout/load.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <TextView
+            android:id="@+id/textView4"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="24dp"
+            android:layout_marginLeft="24dp"
+            android:layout_marginRight="24dp"
+            android:layout_marginStart="24dp"
+            android:layout_marginTop="8dp"
+            android:text="@string/type_paste"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"/>
+
+        <EditText
+            android:id="@+id/etTransaction"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="4dp"
+            android:layout_marginRight="4dp"
+            android:inputType="textNoSuggestions"
+            android:minLines="3"
+            app:layout_constraintEnd_toStartOf="@+id/btnPaste"
+            app:layout_constraintStart_toStartOf="@+id/textView4"
+            app:layout_constraintTop_toBottomOf="@+id/textView4"
+            tools:text="@string/test_address">
+            <requestFocus/>
+        </EditText>
+
+        <!-- Bottom constraint keeps button aligned with text box before API 21 (see dimens.xml). -->
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
+            android:id="@+id/btnPaste"
+            style="@style/FAB.Dialog"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="@+id/etTransaction"
+            app:layout_constraintEnd_toEndOf="@+id/textView4"
+            app:layout_constraintTop_toTopOf="@+id/etTransaction"
+            app:srcCompat="@drawable/ic_copy_24dp"/>
```

Comment:
If it's a paste button then it should use the paste icon, not the copy icon. In Android Studio you can add new icons from the library as follows:
* Right-click the drawable folder and choose New > Vector Asset.
* Click the icon next to the "Clip Art" label, and select the one you want.

Weirdly, the cut, copy and paste icons seem to be missing from Android Studio 4.0, but I can still find them in Android Studio 3.5.

Reply:
My clip art cupboard was barren in Android studio, I added this icon for paste https://material.io/resources/icons/?icon=content_paste&style=baseline

Reply:
Thanks, that's useful to know. For consistency with the existing icons, the filename should be changed to "dp" rather than "px", and the word "content" should be removed.

Reply:
changed filename